core/src/main/java/io/aiven/kafka/tieredstorage/transform/FetchChunkEnumeration.java

```diff
@@ -35,7 +33,7 @@
 
 import org.apache.commons.io.input.BoundedInputStream;
 
-public class FetchChunkEnumeration implements Enumeration<InputStream> {
+public class FetchChunkEnumeration extends CloseableEnumeration<InputStream> {
```

Comment:
Aren't the other enumerations also affected by this issue? If they aren't, maybe we can document on the code why are these proposed wrappers required.